src/responder/kcm/kcmsrv_cmd.c

```diff
@@ -418,6 +430,19 @@ static errno_t kcm_recv_data(int fd, struct kcm_reqbuf *reqbuf)
         return ret;
     }
 
+    msglen = kcm_input_get_payload_len(reqbuf->v_len.kiov_base,
+                                       reqbuf->v_len.kiov_len);
+    msg = talloc_zero_size(mem_ctx, msglen);
```

Comment:
This is totally a personal preference, but I would have `talloc_zero_array(iobuf, uint8_t, msglen);`

Reply:
Suggestion taken, fixed locally, thanks!